syntax = "proto3";

package rpc_package;

service FaceRecognitionService {
    rpc FaceRecognition (RecRequest) returns (RecResponse) {}
}

message RecRequest {
    repeated float feature = 1;
}

message RecResponse {
    string name = 2;
}